src/settings/components/Sets/SetsForm/components/GeneralInformation/GeneralInformation.js

```diff
@@ -13,14 +15,27 @@ import {
 import { ViewMetaData } from '@folio/stripes/smart-components';
 
 import {
+  TEXT_FIELD_MAX_FIELD_LENGTH,
+  TEXT_AREA_FIELD_MAX_FIELD_LENGTH,
   GENERAL_ACCORDION_NAME,
   METADATA_ACCORDION_NAME,
   SET_FIELDS,
 } from '../../../../../constants';
 
+import {
+  validateRequiredField,
+  validateFieldLength,
+  composeValidators,
+} from '../../../../../util';
+
+const validateNameLength = (value) => validateFieldLength(value, TEXT_FIELD_MAX_FIELD_LENGTH);
+const validateDescription = (value) => validateFieldLength(value, TEXT_AREA_FIELD_MAX_FIELD_LENGTH);
```

Comment:
For constistency with the name above.
```suggestion
const validateDescriptionLength = (value) => validateFieldLength(value, TEXT_AREA_FIELD_MAX_FIELD_LENGTH);
```

Reply:
Fixed